spec/fund_america/api_info_spec.rb

```diff
@@ -2,20 +2,18 @@
 
 describe FundAmerica::ApiInfo do
   context '#info' do
-    before(:all) do
-      @response = FundAmerica::ApiInfo.info
-    end
+    let(:response) { FundAmerica::ApiInfo.info }
 
-    it 'must have account_name in response' do
-      expect(@response['account_name']).not_to be nil
+    it 'return proper account_name' do
+      expect(response['account_name']).to eq('BnkToTheFuture')
```

Comment:
So this example will work only when somebody has access to BF API keys. This is a gem, so by definition, it should be usable in other projects by other teams